src/core/router/net_db/impl.cc

```diff
@@ -978,29 +979,23 @@ std::vector<IdentHash> NetDb::GetClosestFloodfills(
   IdentHash dest_key = CreateRoutingKey(destination); {
     std::unique_lock<std::mutex> l(m_FloodfillsMutex);
     for (auto it : m_Floodfills) {
```

Comment:
May as well `const auto&` while you're here.

Reply:
Will do.